src/app/modules/shared-components/components/textarea/textarea.component.ts

```diff
@@ -18,12 +21,16 @@ export class TextareaComponent implements OnInit {
 
   ngOnInit(): void {}
 
+  getValue(): string {
+    return this.parentForm?.get(this.inputName)?.value as string;
+  }
+
   setValue(text: string): void {
```

Comment:
getValue/setValue and change detection should be requested directly to the parentForm I assume.

Reply:
They have been removed